package.json

```diff
@@ -5,7 +5,8 @@
   "main": "index.js",
   "scripts": {
     "benchmark": "cd benchmarks && npm install && npm run all",
-    "test": "standard && tap test.js"
+    "test": "standard && tape test.js",
```

Comment:
we need a job that runs code coverage. Can you add nyc? Thanks

Reply:
Considering there was no job for that before, I’d see it as not related to the current PR. Nevertheless, here’s a commit that adds nyc.